recipes/tipp/meta.yaml

```diff
@@ -18,10 +18,12 @@ requirements:
     - {{ compiler('cxx') }}  # For compiling C++ code
     - make
     - kmc
+    - libzlib >=1.2.13,<1.3.0
```

Comment:
I have not read through your commits. Is this rather strict pinning really necessary or could you relax at least the upper bound?

Reply:
By default, the bioconda-build system was using `libzlib 1.3.1`, which caused issues with dependencies like `minimap2` and `graphaligner`, as these appear to have `libzlib` pinned to an older version. To resolve this, I explicitly pinned the `libzlib` version in the recipe.  

Previously, `libzlib` was not part of the `meta.yaml` file, and when I built it locally using `bioconda-build`, it still worked, as by default it is using`libzlib 1.2.13`. However, after pushing it to the current Bioconda build system, it started using the latest version of `libzlib` (1.3.1), which seems to have been updated very recently. This change triggered the errors, which are resolved by pinning the `libzlib` version.  I believe several other packages have also started encountering the same issue recently.